doc/040_cvarlist.md

```diff
@@ -48,6 +48,12 @@ it's `+set busywait 0` (setting the `busywait` cvar) and `-portable`
 * **aimfix**: Fix aiming. When set to to `0` (the default) aiming is
   slightly inaccurate, bullets and the like have a little drift. When
   set to `1` they hit exactly were the crosshair is.
+  
+* **machinegun_norecoil**: Disable machine gun recoil in single player. 
+  By default this is set to `0`, this keeps the original machine gun 
+  recoil in single player. When set to `1` the recoil is disabled in
+  single player, the same way as in multiplayer.
+  This cvar only works if the game.dll implements this behaviour.
```

Comment:
Can we rename the cvar to `g_machinegun_norecoil`? I know, some of the older cvars are missnamed, but we should do it right for new ones.